packages-exp/auth-exp/index.rn.ts

```diff
@@ -41,7 +41,7 @@ export * from './src';
  *
  * @public
  */
-export const reactNativeLocalPersistence =
+export const reactNativeLocalPersistence: Persistence =
```

Comment:
Need to explicitly import the type here, or else rollup? typescript? transitively gets it from the file `getReactNativePersistence` comes from, and for some reason codes it as an `import()` statement in the `d.ts`, which makes api-extractor error.